_layouts/page.html

```diff
@@ -59,15 +59,40 @@ <h1 class="post-header__title">{{ page.title }}</h1>
 
         {{content}}
 
-        <div class="confirm-block">
-            <strong class="confirm-block__title">
-                Was this article helpful?
-            </strong>
-            <div class="confirm-block__buttons">
-                <a href="#" class="button confirm-block__button button--black">No</a>
-                <a href="#" class="button confirm-block__button button--red">Yes</a>
+        <aside class="form-collapse">
+            <div class="confirm-block">
+                <strong class="confirm-block__title">
+                    Was this article helpful?
+                </strong>
+                <div class="confirm-block__buttons">
+                    <a href="#" class="button confirm-block__button button--black form-collapse__opener" data-feedback="no">No</a>
+                    <a href="#" class="button confirm-block__button button--red form-collapse__opener" data-feedback="yes">Yes</a>
+                </div>
             </div>
-        </div>
+
+            <div class="form-collapse__slide">
+                <form class="feedback-form">
+                    <h3 class="feedback-form__title" data-toc-skip>Please provide feedback</h3>
+                    <div class="feedback-form__block">
+                        <div class="feedback-form__column">
+                            <input type="text" placeholder="Name (Optional)" class="feedback-form__input">
+                        </div>
+                        <div class="feedback-form__column">
+                            <input type="email" placeholder="Email (Optional)" class="feedback-form__input">
+                        </div>
+                        <div class="feedback-form__column feedback-form__column--full">
+                            <textarea placeholder="Any additional feedback?" class="feedback-form__textarea"></textarea>
+                        </div>
+                    </div>
+                    <div class="feedback-form__submit-row">
```

Comment:
Suggest to rename into `feedback-form__action-row` for example, as long as it's not related to submit actually.